adze-web/src/LinksSection.js

```diff
@@ -1,71 +1,122 @@
-import React from 'react'
-
-
+import React, { useState } from 'react'
 import Constants from './Constants.js'
+import { selectManifest, addLinkByUrl, removeLink } from './state/manifestSlice.js'
+import { selectCredentials } from './state/credentialsSlice.js'
+import { useSelector, useDispatch } from 'react-redux'
+import { ErrorMessageOrNull, ManifestStatusMessage } from './notifications.js'
 
-import { selectManifest , addLinkByUrl, removeLink } from './state/manifestSlice.js'
-import { selectCredentials  } from './state/credentialsSlice.js'
-import { useSelector, useDispatch} from 'react-redux'
+const numberOfTags = 4;

```

Comment:
This is more like `numberOfTagsOnInputForm`, yes?  We can limit the UI to just showing 4 tag fields for right now if that's easier, but one goal would be to let people add arbitrary numbers of tags here.